test/functional/legacy/003_cindent_spec.lua

```diff
@@ -0,0 +1,4216 @@
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1
+      				&& 1)) break;
+      }
+      
+      foo()
+      {
+      	a = cond ? foo() : asdf
+      		+ asdf;
+      
+      	a = cond ?
+      		foo() : asdf
+      		+ asdf;
+      }
+      
+      int  main(void)
+      {
+      	if (a)
+      		if (b)
+      			2;
+      		else 3;
+      	next_line_of_code();
+      }
+      
+      barry()
+      {
+      	Foo::Foo (int one,
+      			int two)
+      		: something(4)
+      	{}
+      }
+      
+      barry()
+      {
+      	Foo::Foo (int one, int two)
+      		: something(4)
+      	{}
+      }
+      
+      Constructor::Constructor(int a,
+      		int b 
+      		)  : 
+      	BaseClass(a,
+      			b,
+      			c),
+      	mMember(b)
+      {
+      }
+      int main ()
+      {
+      	if (lala)
+      		do
+      			++(*lolo);
+      		while (lili
+      				&& lele);
+      	lulu;
+      }
+      
+      int main ()
+      {
+      	switch (c)
+      	{
+      		case 'c': if (cond)
+      				  {
+      				  }
+      	}
+      }
+      
+      main()
+      {
+      	(void) MyFancyFuasdfadsfnction(
+      			argument);
+      }
+      
+      main()
+      {
+      	char	foo[] = "/*";
+      	/* as
+      	   df */
+      	hello
+      }
+      
+      /* valid namespaces with normal indent */
+      namespace
+      {
+      	{
+      		111111111111;
+      	}
+      }
+      namespace /* test */
+      {
+      	11111111111111111;
+      }
+      namespace // test
+      {
+      	111111111111111111;
+      }
+      namespace
+      {
+      	111111111111111111;
+      }
+      namespace test
+      {
+      	111111111111111111;
+      }
+      namespace{
+      	111111111111111111;
+      }
+      namespace test{
+      	111111111111111111;
+      }
+      namespace {
+      	111111111111111111;
+      }
+      namespace test {
+      	111111111111111111;
+      	namespace test2 {
+      		22222222222222222;
+      	}
+      }
+      
+      /* invalid namespaces use block indent */
+      namespace test test2 {
+      	111111111111111111111;
+      }
+      namespace11111111111 {
+      	111111111111;
+      }
+      namespace() {
+      	1111111111111;
+      }
+      namespace()
+      {
+      	111111111111111111;
+      }
+      namespace test test2
+      {
+      	1111111111111111111;
+      }
+      namespace111111111
+      {
+      	111111111111111111;
+      }
+      
+      /* end of AUTO */
+      ]=])
+  end)
+
+  it('2 is working', function()
+    insert_([=[
+      
+      {
+      
+      /* this is
+       * a real serious important big
+       * comment
+       */
+      	/* insert " about life, the universe, and the rest" after "serious" */
+      }
+      ]=])
+
+    execute('set tw=0 wm=60 columns=80 noai fo=croq')
```

Comment:
I tried different `'columns'` settings and yes, it does affect the formatting (perhaps together with `'tw'` and `'wm'`?). Not sure it's worth the trouble investigating.